### PR TITLE
Add AX200 Intel chip to BluetoothHost.inf

### PR DESCRIPTION
**Pull request by Shqrp**
Hi chrizonix,

sorry I forgot to put the device in this file, I should've searched better lol.

### CODE REVIEW

**Comment by chrizonix** [comment]
Hi Andrea,

Don't worry, the new ScpDriverInstaller doesn't use the BluetoothHost.inf and BluetoothHost.cat files, for installing a new device.
The reason for this is simple, those files are signed by Nefarius, and changes to these files will invalidate the signature (and driver files).

In the new ScpDriverInstaller (with the option "Use Self-Signed Certificates") I use the WdiWrapper to create a new *.inf file for the target device, on-the-fly (https://github.com/chrizonix/ScpToolkit/blob/master/ScpControl/Driver/DriverInstaller.cs#L29).

Therefore the BluetoothHost.inf file is only used for the original drivers (from Nefarius), that are correctly signed with valid certificates.

So long story short:
Adding a new device is really as simple as adding a new line to `ScpControl.ini` :)

**Comment by Shqrp** [comment]
Ohh, nice! Then I will just compile the installer so that I can install the new drivers, or is there also a local file I can edit directly?

**Comment by chrizonix** [comment]
Yep, you can edit the `ScpControl.ini` directly, without having to compile a new installer.
The file is located here `C:\Program Files\ScpToolkit\ScpControl.ini`.

After editing the file, just launch ScpDriverInstaller and use "Advanced Settings -> Use Self-Signed Drivers" when installing the Bluetooth Devices. That should do the trick :)
